File: CSC2300/Overflow.java
public class Overflow {
    public static void main(String[] args) {
        int a = 2_147_483_637;
        int b = 10;
        System.out.println("The value of a is " + a);
        System.out.println("The value of b is " + b);
	int c = a + b;
	System.out.println("The sum of a and b is " + c);
    }
}

File: CSC2300/overflow.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    unsigned char a = 245;
    unsigned char b = 10;
    unsigned char c;
    printf("The value of a is %u\n", a);
    printf("The value of b is %u\n", b);
    c = a + b;
    printf("The sum of a and b is %u\n", c);
}
